tests/_test_msui/test_msui.py

```diff
@@ -328,3 +328,225 @@ def test_flight_track_io(self, mockload, mocksave, mockq, mocki, mockw):
         assert self.window.listFlightTracks.count() == 2
         assert os.path.exists(self.save_ftml)
         os.remove(self.save_ftml)
+
+class Test_MSUIMainWindow:
+    def setUp(self):
+        # Create a QApplication instance for Qt tests
+        self.app = QtWidgets.QApplication([])
```

Comment:
We use pytest-qt, so there is no need to create another QApplication object. You should use the qtbot fixture instead.